WebKitBrowser/WebKitBrowser.cpp

```diff
@@ -309,6 +309,7 @@ namespace Plugin {
     {
         string message(string("{ \"url\": \"") + URL + string("\", \"loaded\": ") + (loaded ? string("true") : string("false")) + string(" }"));
         TRACE(Trace::Information, (_T("URLChanged: %s"), message.c_str()));
+        _lastURL = URL;
```

Comment:
I would only do the string copy (relatively expensive, certainly for bigger strings) when tracing is enabled for this category (the TRACE_ENABLED(CATEGORY) macro can be used to find that out).

And although I do not see any problem with this PR in itself is what is the actual use case if I may ask? I guess tracing the url that caused Webkit to crash? Or would the error tracing category be always enabled so automated action can be taken?

Reply:
@MFransen69 
The reason for this implementation is, to track the browser crashes by app URL. 
Use case is, We need to know the number of crashes per app basis. 
And we need this log to be printed always.
I can convert (_lastURL =URL) to strcpy.

Reply:
Ah, okay in that case I think you do not want to use Tracing but SYSLOG, correct? Trace lines will only be outputted when for that module and category they are turned on, therefore I suggested to only do the string assignment (which is the same as a strcpy, std::string is not refcounted) when tracing is actually turned on here,  it is off by default. 

But seen your usecase is it perhaps an idea to add this functionality to the PerformanceMetrics plugin, that seems a more logical place then? All the info you need would be available there.
I would be happy to add that functionality there or to assist you in adding it there.